habitat-hitl/habitat_hitl/_internal/networking/keyframe_utils.py

```diff
@@ -45,25 +47,38 @@ def ensure_dict(keyframe, key):
             if not found:
                 consolidated_keyframe["stateUpdates"].append(state_update)
 
-    # if "rigUpdates" in inc_keyframe:
-    #     for rig_update in inc_keyframe["rigUpdates"]:
-    #         id = rig_update["id"]
-    #         pose = rig_update["pose"]
-    #         found = False
-    #         for con_rig_update in consolidated_keyframe["rigUpdates"]:
-    #             if con_rig_update["id"] == id:
-    #                 con_rig_update["pose"] = pose
-    #                 found = True
-    #         if not found:
-    #             consolidated_keyframe["rigUpdates"].append(rig_update)
+    # add or update rigUpdates
+    if "rigUpdates" in inc_keyframe:
+        if networking_config.active_features.skinning:
```

Comment:
This filtering logic shouldn't be needed here. The `skinning` flag isn't going to change at runtime. Why does this code need to do the filtering if you're already going it in HitlDriver?